metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email  'cookbooks@opscode.com'
 license           'Apache 2.0'
 description       'Installs erlang, optionally install GUI tools.'
-version           '1.5.6'
+version           '1.5.7'
```

Comment:
Please remove the version bump. Oftentimes many PRs will be combined and released at once.